src/main/java/com/frcteam3636/frc2024/subsystems/drivetrain/Drivetrain.kt

```diff
@@ -118,6 +123,21 @@ object Drivetrain : Subsystem {
                     gyroRotation.toRotation2d()
                 )
         }
+
+    fun driveWithJoystickPointingTowards(translationJoystick: Joystick, target: Translation2d): Command =
+        run {
+            val magnitude = ROTATION_PID_CONTROLLER.calculate(
```

Comment:
`magnitude` does not appear to be used, and should probably be called something else.